app/models/message_model.py

```diff
@@ -1,19 +1,14 @@
 from dataclasses import dataclass
+from datetime import datetime
 
-from sqlalchemy import Column, DateTime, Integer, String
+from sqlalchemy import Column, DateTime, Integer
 
 from app.configs.database import db
 
 
 @dataclass
 class MessageModel(db.Model):
-    title: str
-    text: str
-    date: str
-
     __tablename__ = "messages"
```

Comment:
Shouldn't we change the name to `message_logs` as well?

Reply:
Done.